server/lib/Cargo.toml

```diff
@@ -26,9 +26,10 @@ name = "image_benches"
 harness = false
 
 [features]
-# default = [ "libsqlite3-sys/bundled", "openssl/vendored" ]
+default = ["dev-oauth2-device-flow"]
```

Comment:
Shouldn't be on by default until it's ready.

Reply:
It's just while I'm testing 😄